README.md

```diff
@@ -361,6 +362,70 @@ To stop receiving events from a stream, pass the `Subscription` object you got w
 client.unsubscribe(sub);
 ```
 
+<a name="data-unions"></a>
+## Data Unions
+This library provides functions for working with Data Unions. To get a DataUnion client instance, call
+`client.dataUnionClient(mainnetAdminPrivateKey, sideChainAdminPrivateKey)`. To deploy a new DataUnion, call
+`dataUnionClient.deployDataUnion(name, adminAddress, adminFeeFractionWei, agents)`. To get an existing instance of
+Data Union call `dataUnionClient.dataUnionFromName(name)`.
+
+### Functions
+
+| Name          | Returns  | Description |
+| :------------ | :------ | :----------- |
+| isDeployed() | boolean | Check if Data Union deployment has completed |
+| addMembers(String ...members) | Transaction receipt | Add members |
+| partMembers(String ...members) | Transaction receipt | Remove members from Data Union |
+| withdrawTokensForSelfOrAsAdmin(String memberAddress, BigInteger amount) | Transaction receipt | Withdraw members tokens to given address |
+| withdrawTokensForMember(BigInteger privateKey, String to, BigInteger amount) | Transaction receipt | Withdraw members tokens |
+
+Here's an example how to deploy a data union contract and set the admin fee:
+
+```java
+DataUnionClient dataUnionClient = new StreamrClient(new StreamrClientOptions()).dataUnionClient("mainnetAdminPrvKey", "sidechainAdminPrvKey");
+BigInteger adminFee = BigInteger.TWO;
+List<String> agents = Arrays.asList("<private key>");
+DataUnion dataUnion = dataUnionClient.deployDataUnion("Cool Data Union", adminAddress, adminFee, agents);
+```
+
+### Member functions
+
+| Name                                  | Returns  |
+| :------------------------------------ | :------- |
+| isMemberActive(String memberAddress)         | boolean  |
+| isMemberInactive(String memberAddress)       | boolean  |
+| withdrawAllTokensForMember(String from, String to, byte[] signedWithdrawalRequest) | Transaction receipt |
```

Comment:
`Join` and possibly the sign withdrawal methods should be added
https://github.com/streamr-dev/streamr-client-javascript#member-functions